src/views/Room.vue

```diff
@@ -300,6 +307,11 @@ export default {
         this.$refs.youtube.player.seekTo(newPosition);
       }
     },
+    inputAddUrlText() {
+      // HACK: The @change event only triggers when the text field is defocused.
+      // This ensures that onInputAddChange() runs everytime the text field's value changes.
+      this.onInputAddChange();
+    },
```

Comment:
See @input event on `v-text-fields`

Reply:
Actually, @keydown might be what I was thinking of.

Reply:
I don't think `@keydown` would be triggered if the user pastes text using the right click context menu. This method ensures we get all changes to the text field, regardless to the source of the change.